src/test/java/com/jcabi/aether/RootArtifactTest.java

```diff
@@ -86,6 +91,32 @@ public void resolvesRootArtifact() throws Exception {
     }
 
     /**
+     * RootArtifact can resolve a parent artifact.
```

Comment:
@prondzyn let's explicitly state the relation between `jcabi-aether` and the parent here, by mentioning the connection between those two artifacts and versions.